src/Libraries/CoreNodesUI/Input/SliderBase.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace DSCoreNodesUI.Input
+{
+    public abstract class SliderBase<T> : BasicInteractive<T>
+    {
```

Comment:
Consider putting a comparable constraint on T, and doing the if min > max stuff here?